src/archutils/Win32/StepMania.in.manifest

```diff
@@ -4,6 +4,7 @@
   <application xmlns="urn:schemas-microsoft-com:asm.v3">
     <windowsSettings>
       <activeCodePage xmlns="http://schemas.microsoft.com/SMI/2019/WindowsSettings">UTF-8</activeCodePage>
+      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/pm</dpiAware>
```

Comment:
Sorry I meant to add that comment here.

Reply:
If the setting is not present, it's the same as setting it to `false` which is what causes the original issue that was reported.

Also as I've explained in the commit message, ITGmania is not aware of any DPI settings at all. But when the setting is `false` or missing, there is automatic scaling happening on Windows side which causes issues (like part of the screen being cut off). I think it's better to ignore the settings than to have broken image which is the reason for this fix.